pkg/testutil/testutil.go

```diff
@@ -604,6 +604,12 @@ func RequireExecutable(t testing.TB, name string) {
 	}
 }
 
+func RequireIPFS(t *testing.T) {
```

Comment:
nit: maybe this function can be replaced by just using `RequireExecutable("ipfs")` above.